std/math.d

```diff
@@ -4115,19 +4115,38 @@ real cbrt(real x) @trusted nothrow @nogc
  *      )
  */
 real fabs(real x) @safe pure nothrow @nogc { pragma(inline, true); return core.math.fabs(x); }
-//FIXME
+
 ///ditto
-double fabs(double x) @safe pure nothrow @nogc { return fabs(cast(real) x); }
-//FIXME
+pragma(inline, true)
+double fabs(double d) @trusted pure nothrow @nogc
+{
+    ulong tmp = *cast(ulong*)&d & 0x7FFF_FFFF_FFFF_FFFF;
+    return *cast(double*)&tmp;
```

Comment:
is this Endian safe?

Reply:
It should be fine. I added unittests to verify.

Reply:
The CI wouldn't really catch that though, as it's all LittleEndian.

However... FYI @thewilsonator, all targets I'm aware use _same_ endianness for both float and integer words, regardless whether the it's a BigEndian or LittleEndian architecture.  The one target that is the exception to the rule is the PDP-11, and I doubt Phobos will ever support that. :-)